controllers/gateway/api_controller_integration_test.go

```diff
@@ -43,6 +44,127 @@ var _ = Describe("APIRule Controller", Serial, func() {
 		defaultHttpTimeout         = time.Second * 180
 	)
 
+	Context("check default domain logic", func() {
+
+		It("should have an error when creating an APIRule without a domain in cluster without kyma-gateway", func() {
+			updateJwtHandlerTo(helpers.JWT_HANDLER_ORY)
```

Comment:
I think it would be better to use Istio instead of Ory, if we want to have a test only for one of this handlers, since Istio is the handler that will stay.
If we drop the Ory handler we'd have to adapt this test for Istio and in worst-case it will get deleted, because it's an Ory test.